code/script.js

```diff
@@ -28,24 +86,343 @@ const showMessage = (message, sender) => {
+    setTimeout(() => showMessage(`Amazing! What genre would you be interested in? 🍿`, "bot"), 2000); // An answer from the bot, with a follow-up question
+    btnOne.remove(); // Yes-button gets removed to make room for the select with options that comes further down
+    btnTwo.remove(); // No-button gets removed to make room for the select with options that comes further down
+    showTypingIndicator(); // Display the typing indicator
+    setTimeout(() => removeTypingIndicator(), 2000);
+    setTimeout(() => chooseGenre(), 1999) // Here I put 999 milliseconds, otherwise the selectdropdown was palced above the text to choose genre. This needed to come just a millisecond before for everything to look good!
+  })
+
+  btnTwo.addEventListener("click", () => {
+    showMessage("No 👎", "user");
+    setTimeout(() => showMessage(`I understand, I hope you'll have an amazing night!`, "bot"), 1000);
+    btnOne.remove();
+    btnTwo.remove();
+    showTypingIndicator(); // Display the typing indicator
+    setTimeout(() => removeTypingIndicator(), 2000);
+    setTimeout(() => endChat(), 2000)
+  })
+};
+
+/* ------------- FOURTH INTERACTION ------------- */
+// Creates the select dropdown with different genre-options
+const chooseGenre = () => {
+  createSelectMenu("Action", "Comedy", "Drama");
+  // Declares a varaible for the select element to be used when getting the text content of it further down
+  const selectDropDown = document.querySelector('.select-menu');
+  chat.scrollTop = chat.scrollHeight; // Scroll to the bottom to show the indicator
+
+  // Function to check what genre the user has chosen.
+  const checkValue = () => {
+    // This variable needs to be inside of the checkValue function, if it's outside of it, the code hasn't had time to "store" the value yet, and therefore it returns an error
+    const genre = selectDropDown[selectDropDown.selectedIndex].text;
+    selectedGenre = genre; // selectedGenre is the same as genre
+
+    const optionInteraction = () => {
+      showMessage(`${genre}`, "user");
+      showTypingIndicator(); // Display the typing indicator
+      setTimeout(() => removeTypingIndicator(), 2000);
+      setTimeout(() => showMessage(`${genre} is a great choice! Would you like to see an oldie but a goodie, or a newer movie?`, "bot"), 2000);
+      selectDropDown.value = " ";
+      setTimeout(() => selectDropDown.remove(), 2000);
+    }
+
+    optionInteraction();
+    setTimeout(() => createButtons("🧓🏻", "btn-one", "🍼", "btn-two"), 2000);
+    setTimeout(() => oldieOrNewbie(), 2000);
+  }
+  selectDropDown.addEventListener("change", checkValue); // on the change of a select option, the code for checkValue is invoked.
+
+};
+
+/* ------------- FIFTH INTERACTION ------------- */
+// Lets user select from an old movie or a new movie
+function oldieOrNewbie() {
+  const oldie = document.getElementById("btn-one");
+  const newbie = document.getElementById("btn-two");
```

Comment:
What do you think about changing this function to the one with arrows to use the same "method" throughout the js? 
```
const oldieOrNewbie = () => {
  const oldie = document.getElementById("btn-one");
  const newbie = document.getElementById("btn-two");
```